Makefile

```diff
@@ -15,6 +15,16 @@ sim-test:
 sim-clean:
 	make -C $(SIM_DIR) clean clean-testlog
 
+#
+# CONSOLE
```

Comment:
console is not run  from top makefile 

it is always run implicitly 

please remove

Reply:
make fpga-run triggers console automatically

Reply:
Okay, removed.